packages/web-api/src/controllers/scan-request-controller.ts

```diff
@@ -174,6 +175,11 @@ export class ScanRequestController extends ApiController {
             return { valid: false, error: WebApiErrorCodes.missingSiteOrReportGroups.error };
         }
 
+        const validReportGroup = scanRunRequest?.reportGroups?.find((g) => !isEmpty(g?.consolidatedId));
```

Comment:
this should be the same check like the consolidated report (check for both urls and valid reportgroup), but if the deepscan is enabled return the new error